zetta_utils/training/lightning/train.py

```diff
@@ -267,13 +293,31 @@ def _lightning_train_remote(
     Creates a volume mount for `train.cue` in `/opt/zetta_utils/specs`.
     Runs the command `zetta run specs/train.cue` on one or more worker pods.
     """
+    if train_args["trainer"]["accelerator"] == "gpu":
```

Comment:
The [[example DDP spec]](https://github.com/ZettaAI/zetta_utils/blob/9ddf8818fede27a119f5912ba17ecbfcc1220a40/specs/examples/training/ddp.cue#L63) sets `accelerator` to "cuda", resulting in a `NotImplementedError`.